packages/oscal-react-library/src/components/OSCALCatalogGroup.tsx

```diff
@@ -324,6 +402,11 @@ const OSCALCatalogGroupList: React.FC<OSCALCatalogGroupListProps> = (props) => {
       previousHandledFragment={previousHandledFragment}
       setPreviousHandledFragment={setPreviousHandledFragment}
     >
+      {group.parts
+        ?.map((groupPart: Part) => groupPart.prose)
```

Comment:
I am curious why we need to specify the type here -- `tsc` should be able to infer it right?